README.adoc

```diff
@@ -445,9 +450,12 @@ guide.
 
 [role="code_command hotspot file=0", subs="quotes"]
 ----
-#Create the `kubernetes.yaml`.#
+#Update the `kubernetes.yaml`.#
 `kubernetes.yaml`
 ----
+[role="edit_command_text"]
+Replace [hotspot=sysImage hotspot=invImage file=0]`[registry-server]` with your container registry server.
+You can get the login server for your registry by running `az acr show -n [registry-name] --query loginServer` command.
```

Comment:
Update to
"You can get the login server for your registry by running the `az acr show -n [registry-name] --query loginServer` command."